City-Bureau/city-scrapers
# 🏗️ Build spider: Illinois African Descent Citizens Reparations Commission

## What's this PR do?
Adds a spider to scrape meetings from the website of Illinois African Descent Citizens Reparations Commission. The new spider is called `il_adcrc`.

## Why are we doing this?
Requested by City Bureau's Chicago office.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl il_adcrc -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here](https://spokanevalley.granicus.com/ViewPublisher.php?view_id=3).

## Are there any smells or added technical debt to note?
